afterpay/src/main/kotlin/com/afterpay/android/view/AfterpayPriceBreakdown.kt

```diff
@@ -251,26 +257,40 @@ class AfterpayPriceBreakdown @JvmOverloads constructor(
 
     private fun generateContent(afterpay: AfterpayInstalment): Content = when (afterpay) {
         is AfterpayInstalment.Available -> {
-            val template: AfterpayOptionalText = if (showInterestFreeText && showWithText) {
-                AfterpayOptionalText.INTEREST_FREE_AND_WITH
-            } else if (showInterestFreeText) {
-                AfterpayOptionalText.INTEREST_FREE_AND_WITH
-            } else if (showWithText) {
-                AfterpayOptionalText.WITH
-            } else {
-                AfterpayOptionalText.NONE
+            val isUkLocale = Afterpay.configuration?.locale == Locales.EN_GB
+            val isGbpCurrency = Afterpay.configuration?.currency == Currency.getInstance(Locales.EN_GB)
+
+            val withText: String = when {
+                showWithText -> Afterpay.strings.priceBreakdownWith
+                else -> ""
             }
 
+            val interestFreeText: String = when {
+                isUkLocale || isGbpCurrency -> ""
+                showInterestFreeText -> Afterpay.strings.priceBreakdownInterestFree
+                else -> ""
+            }
+
+            val numberOfInstalments: Int = Afterpay.configuration?.let {
+                AfterpayInstalment.numberOfInstalments(it.currency)
```

Comment:
What's `it`? What does this line do?

Reply:
Kotlin feature: [scope block](https://kotlinlang.org/docs/scope-functions.html)

In this case, `it` refers to `Afterpay.configuration` and can be seen in the ide

<img width="611" alt="Screen Shot 2022-07-05 at 13 52 52" src="https://user-images.githubusercontent.com/76413218/177246202-c89251b8-bca3-4635-8833-4494c51a8676.png">